### PR TITLE
Split all companies out into separate files

### PR DESCRIPTION
**Pull request by drewbrew**
Supersedes #845

### CODE REVIEW

**Comment by drewbrew** [comment]
CC @toothbrush

**Comment by toothbrush** [comment]
So i don't know if you care about it, but @con-cat has just been working on a small improvement to the script so that it grabs the `git blame` of the original entry and credits them in the new commit, making the person who runs `convert.rb` merely the `Committer`, not the `Author`.  What do you think?

File: companies.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ Go [Home](/). Jump to: <a href="/events.html">Events</a>. Jump to: <a href="/uni
 
 <a name="companies"></a>
 ## Companies - {{ site.data.companies | size }}
-{% assign sorted = site.data.companies | sort_natural: 'name' %}
+{% assign sorted = site.data.companies | sort %}
```
Comment (toothbrush) [inline companies.md]: If `sort_natural` still works, we'd probably want to keep that.

Comment (drewbrew) [reply companies.md]: I'll give it a shot. Still getting Ruby set up locally.

**Comment by toothbrush** [comment]
> > So i don't know if you care about it, but @con-cat has just been working on a small improvement to the script so that it grabs the `git blame` of the original entry and credits them in the new commit, making the person who runs `convert.rb` merely the `Committer`, not the `Author`
> 
> I'm quite happy not taking the blame!

Cool, give us 20 min to fold it into the script.  Will be back.

**Comment by drewbrew** [comment]
Knowing my luck, it'll take that long for the ruby installs to finish!

File: companies.md (hunk 1 above)
Comment (drewbrew) [reply companies.md]: image
Looks like `sort_natural` does not work.

**Comment by toothbrush** [comment]
Haha, we're just preparing a branch.

**Comment by drewbrew** [comment]
I've yanked all the migrating from this PR so we can have a "clean" branch to work from. I figure I can pull your newest changes here and then go.

**Comment by drewbrew** [comment]
I'm about to doze off (it's pushing 10:30 pm my time), so I'm signing off for the night. If @phildini wants to take over, go for it. Otherwise, I'll tackle the migration tomorrow US time.

**Comment by toothbrush** [comment]
> I've yanked all the migrating from this PR so we can have a "clean" branch to work from. I figure I can pull your newest changes here and then go.

Uh oh, we'd already done the same 😅 Oh well!

**Comment by toothbrush** [comment]
> I've yanked all the migrating from this PR so we can have a "clean" branch to work from. I figure I can pull your newest changes here and then go.

I think it's easier, with big refactors like this, if we don't simultaneously try and tackle it.  I propose closing this PR in favour of #864.

File: companies.md (hunk 1 above)
Comment (toothbrush) [reply companies.md]: Yep, i had found that one. 
 Addressed in https://github.com/phildini/stayinghomeclub/pull/864/commits/b3ea5ead5744d06fac0c19cdfe431a194c6f8bbb.

**Comment by phildini** [comment]
Superseded by #864

**Pull request closed without merging**